jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -101,11 +101,11 @@ def buildExclusions = [
   [ /sharedlibs_shared/,              anyType,     lt(9)   ],
 
   // OSX ---------------------------------------------------
-  [ /^osx1010/,                       anyType,     gte(11) ],
-  [ /^osx1011/,                       releaseType, lt(11)  ],
-  [ /^osx1011/,                       releaseType, gte(13) ],
+  [ /^osx1010/,                       testType,    gte(11) ],
+  [ /^osx1010/,                       releaseType, noVer   ],
   [ /^osx1011/,                       testType,    gte(14) ],
-  [ /^osx1015/,                       releaseType, lt(13)  ],
+  [ /^osx1011/,                       releaseType, noVer   ],
+  [ /^osx1015/,                       releaseType, allVer  ],
```

Comment:
```suggestion
  [ /^osx1015/,                       releaseType, noVer  ],
```

Exclude 10.15 for release build on "no versions" -- which is a no-op, it should just be removed.